pennylane/devices/tests/conftest.py

```diff
@@ -262,3 +262,60 @@ def pytest_runtest_makereport(item, call):
                 tr.outcome = "skipped"
 
     return tr
+
+
+@pytest.fixture
+def in_tape_mode():
+    """Run the test in tape mode"""
+    qml.enable_tape()
+    yield
+    qml.disable_tape()

```

Comment:
Is this ever used in `devices/tests`? :thinking:

Reply:
Nope, good catch! Removed.